cypress/integration/trust-admin/editingAHospital.js

```diff
@@ -0,0 +1,82 @@
+describe("As an admin, I want to edit a hospital so that I can keep hospital changes up to date.", () => {
+  beforeEach(() => {
```

Comment:
Do we need to run this setup before each test in this file? Ideally we want to run this as little as possible to reduce the runtime of the e2e tests

Reply:
If I don't then `ThenIShouldBeOnTheEditHospitalPage` doesn't work the second time round. I can update the tests to work around this, if you'd prefer?

Reply:
Ah in that case what if we re-ordered the specs so that we didn't have to run the setup twice ?

Reply:
Good idea, wish I'd thought of that.